src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Xamarin.Android.Build.Tests.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\..\packages\NUnit3TestAdapter.3.12.0\build\net35\NUnit3TestAdapter.props" Condition="Exists('..\..\..\..\packages\NUnit3TestAdapter.3.12.0\build\net35\NUnit3TestAdapter.props')" />
```

Comment:
This is the thingy that makes the test runner UI in Visual Studio 2019 work.

Reply:
As per `packages.config`, `NUnit3TestAdapter` is a `net472` package.  Will this work if .NET 4.7.2 isn't available?  Or are we saying that we *will* have .NET 4.7.2 available *once `make prepare` finishes*, and thus it'll work?